ferry/transform/import_courses.py

```
@@ -132,14 +131,14 @@ def resolve_cross_listings(listings: pd.DataFrame) -> tuple[pd.DataFrame, pd.Dat
         # Skip if this season is not in the current dataset
         if season not in listings["season_code"].values:
             continue
-            
+
         first, *rest = crns
```

Ruff (0.15.15)

[warning] Unpacked variable `first` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)